### PR TITLE
don't prompt for automatic tasks

### LINKED ISSUE #160174: Task doesn't run on folder open anymore
* https://github.com/microsoft/vscode-github-issue-notebooks
* notice that the `task.json` has a `"runOn": "folderOpen"`-script
* run the task manually
* reload window
* :bug: the task doesn't start automatically

### PR DESCRIPTION
**Pull request by meganrogge**
fix #160174

### CODE REVIEW

File: src/vs/workbench/contrib/tasks/browser/task.contribution.ts [hunk 1]
```diff
@@ -495,15 +495,9 @@ configurationRegistry.registerConfiguration({
...
-			enumDescriptions: [
-				nls.localize('task.allowAutomaticTasks.on', "Always"),
-				nls.localize('task.allowAutomaticTasks.auto', "Prompt for permission for each folder"),
-				nls.localize('task.allowAutomaticTasks.off', "Never"),
-			],
-			description: nls.localize('task.allowAutomaticTasks', "Enable automatic tasks in the folder - note that tasks won't run in an untrusted workspace."),
-			default: 'auto',
+			type: 'boolean',
+			description: nls.localize('task.allowAutomaticTasks', "Enable automatic tasks in any trusted workspace"),
+			default: true,
```
Comment (Tyriar) [inline src/vs/workbench/contrib/tasks/browser/task.contribution.ts:500]: I thought we'd want to still keep auto, just change the default to on?

Comment (meganrogge) [reply src/vs/workbench/contrib/tasks/browser/task.contribution.ts:500]: Oh my thought was to get rid of the prompt entirely

Comment (Tyriar) [reply src/vs/workbench/contrib/tasks/browser/task.contribution.ts:500]: The poll we did was fairly inconclusive. I would still use off, but I suspect there are those who would still want it to be a per-workspace decision. Removing the previous default outright is quite risky

Comment (Tyriar) [reply src/vs/workbench/contrib/tasks/browser/task.contribution.ts:500]: We should also consider `scope: ConfigurationScope.RESOURCE` if we want it to be team choice vs user choice, and how that would interact with auto

**Comment by meganrogge** [comment]
we decided this should be discussed in the UX sync next week

**Pull request closed without merging**